docker-compose.yaml

```diff
@@ -409,7 +414,7 @@ services:
       - MAILJET_API_KEY=${MAILJET_API_KEY:-dummyMailjetApiKey}
       - MAILJET_API_SECRET_KEY=${MAILJET_API_SECRET_KEY:-dummyMailjetApiSecretKey}
       - MAX_FILE_SIZE_MB=80
-      - OFFICE_CONVERTER_CLIENT_MP_REST_URL=http://localhost:9999 # dummy for now
+      - OFFICE_CONVERTER_CLIENT_MP_REST_URL=http://localhost:8083
```

Comment:
`localhost` gaat niet werken want als je ZAC in Docker draait dan kan ie niet bij localhost. Zo werkt Docker.

Maar dat is ook niet nodig. Dit is puur een 'interne' service binnen het Docker Compose netwerk dus volgens mij kan je hier gewoon `http://office-converter:8080` doen zoals we bij de andere variabelen ook doen?